inputs/worker_inner:6.json

```diff
@@ -102,19 +86,13 @@
     {
```

Comment:
`pulp_push` needs `"publish": false` to avoid double publishing content

Reply:
Fixed.